src/description.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var s = require('util').format;
+var marked = require('marked');
```

Comment:
> We need a way to test whether it's a String or a file prior to hit the filesystem IMO. (even dumb)

`readFile`, if error, use description as is (string), otherwise use file contents.